Libplanet/Net/NetMQTransport.cs

```diff
@@ -893,19 +893,26 @@ private async Task CreatePermission(BoundPeer peer)
 
             foreach (IPAddress ip in ips)
             {
-                var ep = new IPEndPoint(ip, peer.EndPoint.Port);
-                if (IPAddress.IsLoopback(ip))
+                try
                 {
-                    // This translation is only used in test case because a
-                    // seed node exposes loopback address as public address to
-                    // other node in test case
-                    ep = await _turnClient.GetMappedAddressAsync(cts.Token);
-                }
+                    var ep = new IPEndPoint(ip, peer.EndPoint.Port);
+                    if (IPAddress.IsLoopback(ip))
+                    {
+                        // This translation is only used in test case because a
+                        // seed node exposes loopback address as public address to
+                        // other node in test case
+                        ep = await _turnClient.GetMappedAddressAsync(cts.Token);
+                    }
 
-                // FIXME Can we really ignore IPv6 case?
-                if (ip.AddressFamily.Equals(AddressFamily.InterNetwork))
+                    // FIXME Can we really ignore IPv6 case?
+                    if (ip.AddressFamily.Equals(AddressFamily.InterNetwork))
+                    {
+                        await _turnClient.CreatePermissionAsync(ep, cts.Token);
+                    }
+                }
+                catch (TaskCanceledException)
                 {
-                    await _turnClient.CreatePermissionAsync(ep, cts.Token);
+                    _logger.Debug($"Task is cancelled during {nameof(CreatePermission)}");
```

Comment:
I guess it would be better we add the filter judges the exception from `cts` like as below...

```csharp
if (cts.IsCancellationRequested)
{
    _logger.Debug($"Task is cancelled during {nameof(CreatePermission)}");
}

throw;
```